assets/js/googlesitekit/widgets/components/WidgetRenderer.js

```diff
@@ -43,7 +44,7 @@ const WidgetRenderer = ( { slug, gridClassName, OverrideComponent } ) => {
 	const widgetComponentProps = useMemo( () => getWidgetComponentProps( slug ), [ slug ] );
 
 	if ( ! widget ) {
-		return null;
+		return <WidgetNull widgetSlug={ slug } />;
```

Comment:
This widget is already provided in `widgetComponentProps`, so no need to re-import it here and pass the slug, since this is already passed as well.

Reply:
This is already resolved.